src/combinators/Divider.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: GPL-v3.0
+
+pragma solidity ^0.8.17;
+
+import '../Feedbase.sol';
+import { Ward } from '../mixin/ward.sol';
+
+contract Divider is Ward {
+    struct Config {
+        address[] sources;
```

Comment:
might help if there was an array of precisions too so all sources don't have to match